<ui version="4.0" >
 <class>FindDialog</class>
 <widget class="QDialog" name="FindDialog" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Find</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>20</y>
     <width>487</width>
     <height>160</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <property name="leftMargin" >
     <number>3</number>
    </property>
    <property name="topMargin" >
     <number>3</number>
    </property>
    <property name="rightMargin" >
     <number>3</number>
    </property>
    <property name="bottomMargin" >
     <number>3</number>
    </property>
    <item row="0" column="0" >
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string>Find:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2" >
     <widget class="QLineEdit" name="lineEdit" />
    </item>
    <item row="1" column="0" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1" >
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QCheckBox" name="checkBox_case" >
        <property name="text" >
         <string>Case sensitive</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_words" >
        <property name="text" >
         <string>Whole words</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_backwards" >
        <property name="text" >
         <string>Find backwards</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_regex" >
        <property name="text" >
         <string>Regular expression</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>281</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0" colspan="3" >
     <widget class="QDialogButtonBox" name="buttonBox" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons" >
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FindDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FindDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
